index.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+
+const app = express();
+const port = 2000;
+
+app.get('/', (req,res)=>{
+    res.send('Hello!!');
+})
+
+app.listen(port, ()=> {
+    console.log(`Listening on port ${port}!!`);
```

Comment:
`console.log` should not be the part of the real time app. Use `log4js` npm package instead.